atlasdb-config/src/test/java/com/palantir/atlasdb/config/AtlasDbConfigsTest.java

```diff
@@ -28,11 +30,11 @@
     private static String previousKeyPathProperty;
 
     @BeforeClass
-    public static void setUpClass() {
+    public static void setUpClass() throws URISyntaxException {
         previousKeyPathProperty = System.getProperty(KeyPair.KEY_PATH_PROPERTY);
         System.setProperty(
                 KeyPair.KEY_PATH_PROPERTY,
-                AtlasDbConfigsTest.class.getResource("/test.key").getPath());
```

Comment:
without using the URI, accessing resources this way produces a path like 
"/C:\Foobar\BazCamp" which breaks the magic that Java has to Windows-ize paths from unix-y versions to Windows-y versions, because it's a little bit of both in a bad way.